iosHackthonApp/Modules/Features/AddMeal/Sources/AddMeal/AddMealView.swift

```diff
@@ -16,114 +16,38 @@ import Components
 public struct AddMealView: View {
 
     public init(viewModel: AddMealViewModel) {
-        self.viewModel = viewModel
+        _viewModel = StateObject(wrappedValue: viewModel)
```

Comment:
Hmmmmm is this definitely needed? I've never had to access the generated `_` property before

Reply:
It does seem a bit hacky, doesn't it.
I can change the viewModel to initialiase the SDK and LocationManager, rather than passing them through and then I _think_ I don't need to do this. I'll check it out